YIF.Core.Domain/ServiceInterfaces/ISuperAdminService.cs

```diff
@@ -22,5 +22,6 @@ public interface ISuperAdminService
         Task<ResponseApiModel<DescriptionResponseApiModel>> UpdateSpecialtyById(SpecialtyPutApiModel model);
         Task<ResponseApiModel<DescriptionResponseApiModel>> AddSpecialtyToTheListOfAllSpecialties(SpecialtyPostApiModel specialityPostApiModel);
         Task<ResponseApiModel<IEnumerable<IoEModeratorsForSuperAdminResponseApiModel>>> GetIoEModeratorsByIoEId(string ioEId);
+        Task<ResponseApiModel<DescriptionResponseApiModel>> DisableInstitutionOfEducation(string adminId);
```

Comment:
rename pls method name on ChangeBannedStatusOfIoE maybe?

Reply:
We have ban/unban, so this method is more BanIoE, or DisableIoE, than ChangeStatus.

Reply:
Good point, we not just ban, we can also unban with this method.

Reply:
yeah, I've changed it